Rian501/gleanTennessee
# El submit screen

The user can either sign in or register and is immediately 'redirected' (unhidden) to a submit request screen which submits their input along with their profile details (whether just created or stored from login). This eliminates the need for separate pages for sign up or glean submission.

This uses formspree to generate and send email.

This is now basically functional.

To test:
1. Clone repo or fetch branch.
1. Register as a new user and submit a glean request.
1. refresh and sign in again as same user and submit another request.
1. If you have access, check the email address currently set to receive these (or, replace the email in the index file with your own for extensive testing)
